crates/module-system/sov-modules-api/src/state/accessors/http_api.rs

```diff
@@ -160,7 +162,7 @@ pub struct ApiStateAccessor<S: Spec> {
     gas_price: <S::Gas as Gas>::Price,
     kernel_cache: ProvableStorageCache<namespaces::Kernel>,
     user_cache: ProvableStorageCache<namespaces::User>,
-    accessory_writes: HashMap<SlotKey, Option<SlotValue>>,
+    accessory_writes: HashMap<SlotKey, (u64, Option<SlotValue>)>,
```

Comment:
Let's add a comment explaining the meaning of the u64 in the tuple

Reply:
Replaced this with a little helper struct; should be much clearer now